buildspec_integ.yml

```diff
@@ -1,10 +1,15 @@
 version: 0.2
 
 phases:
+  install:
+    runtime-versions:
+      golang: 1.13
+  pre_build:
+    commands:
+      - mkdir -p /go/src/github.com/aws/amazon-ecs-cli
+      - cp -R . /go/src/github.com/aws/amazon-ecs-cli/
+      - cd /go/src/github.com/aws/amazon-ecs-cli/
```

Comment:
why do we need to make these directories when we are just going to run the test binary below?

Reply:
I fought with codebuild for quite a long time, and this was the only way I could get the make target to run properly--otherwise I was getting errors where go could not find the dependencies, could not find the test coverage directories for the call to gocovmerge, was failing due to type mismatches stemming from the go dependencies being in a different directory...

Reply:
lol ok